roles/zuul/tasks/main.yml

```diff
@@ -164,6 +165,53 @@
     - Reload systemd units
     - Restart zuul
 
+- name: Create zuul status lib dir
+  file:
+    path: /opt/source/zuul/etc/status/public_html/lib
+    state: directory
```

Comment:
I agree that this is a bit ugly/hacky, but if it makes you feel any better infra does basically the same thing and the public_html/lib directory is in a .gitignore file.